.github/workflows/release-please.yml

```diff
@@ -51,6 +51,24 @@ jobs:
             **/*.exe
             **/*.snap
             **/*.blockmap
+  publish-to-snapcraft:
+    name: Publish to Snapcraft
+    needs:
+      - publish
+      - release-please
+    runs-on: ubuntu-latest
+    steps:
+      - name: Download artifact
+        uses: actions/download-artifact@v3
+        with:
+          name: built-app-x64-linux
+      - name: Publish to Snapcraft
+        uses: snapcore/action-publish@v1
+        env:
+          SNAPCRAFT_STORE_CREDENTIALS: ${{ secrets.SNAPCRAFT_STORE_LOGIN }}
+        with:
+          snap: "*.snap"
+          release: stable
```

Comment:
I have used the Snapcraft official way of building the Snap package before as you can see [here](https://github.com/zidoro/pomatez/tree/master/snap). It triggers when there is a new version released and automatically uploads to the SnapStore but in the `edge` channel which is ideal then I promote the latest version on the edge to the other channels once I tested it on a Linux machine it still worked as expected.

But the Snapcraft GitHub integration has been broken and I don't know what happened to why new builds are failing, maybe it is due to some breaking changes on their end that I failed to be updated also, so I decided to remove the integration though I didn't delete the folder yet cause I was thinking it might still be useful for future reference. Right now, I just manually download the snap package here and upload it manually on SnapStore.